Testsuite: oracle.json.tests.soda.test_OracleCollectionAdmin
Tests run: 32, Failures: 1, Errors: 3, Time elapsed: 141.774 sec

Testcase: testJsonSearchIndex(oracle.json.tests.soda.test_OracleCollectionAdmin):	Caused an ERROR
null
oracle.soda.rdbms.impl.SODAUtils$1OracleSQLException
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:124)
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:87)
	at oracle.soda.rdbms.impl.TableCollectionImpl.insertAndGet(TableCollectionImpl.java:1033)
	at oracle.soda.rdbms.impl.TableCollectionImpl.insertAndGet(TableCollectionImpl.java:734)
	at oracle.soda.rdbms.impl.TableCollectionImpl.insert(TableCollectionImpl.java:724)
	at oracle.json.tests.soda.test_OracleCollectionAdmin.testJsonSearchIndexWithCol(test_OracleCollectionAdmin.java:550)
	at oracle.json.tests.soda.test_OracleCollectionAdmin.testJsonSearchIndex(test_OracleCollectionAdmin.java:698)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at oracle.json.testharness.JsonTestCase.run(JsonTestCase.java:45)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-01400: cannot insert NULL into ("SCOTT"."testIndexAll1"."RESID")

https://docs.oracle.com/error-help/db/ora-01400/
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:608)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1277)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1102)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:456)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:482)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:259)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:1207)
	at oracle.jdbc.driver.OracleStatement.executeSQLStatement(OracleStatement.java:1652)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1417)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3731)
	at oracle.jdbc.driver.OraclePreparedStatement.executeLargeUpdate(OraclePreparedStatement.java:3912)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3887)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:993)
	at oracle.soda.rdbms.impl.TableCollectionImpl.insertAndGet(TableCollectionImpl.java:955)
Caused by: Error : 1400, Position : 10, SQL = insert into "SCOTT"."testIndexAll1" ("DATA") values (:1 ) returning "RESID", "ETAG" into :2 , :3 , Original SQL = insert into "SCOTT"."testIndexAll1" ("DATA") values (?) returning "RESID", "ETAG" into ?, ?, Error Message = ORA-01400: cannot insert NULL into ("SCOTT"."testIndexAll1"."RESID")

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:710)


Testcase: testCreateIndexTTL(oracle.json.tests.soda.test_OracleCollectionAdmin):	Caused an ERROR
null
oracle.soda.rdbms.impl.SODAUtils$1OracleSQLException
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:124)
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:87)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.callCreateIndexPLSQL(OracleCollectionImpl.java:1789)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.createIndex(OracleCollectionImpl.java:1959)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.access$900(OracleCollectionImpl.java:69)
	at oracle.soda.rdbms.impl.OracleCollectionImpl$OracleCollectionAdministrationImpl.createIndex(OracleCollectionImpl.java:2398)
	at oracle.json.tests.soda.test_OracleCollectionAdmin.testCreateIndexTTL(test_OracleCollectionAdmin.java:2300)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at oracle.json.testharness.JsonTestCase.run(JsonTestCase.java:45)
Caused by: java.sql.SQLException: ORA-27486: insufficient privileges

https://docs.oracle.com/error-help/db/ora-27486/
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:608)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1277)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1102)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:456)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:482)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:216)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:1164)
	at oracle.jdbc.driver.OracleStatement.executeSQLStatement(OracleStatement.java:1652)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1417)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3731)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3945)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4202)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.execute(OraclePreparedStatementWrapper.java:1011)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.callCreateIndexPLSQL(OracleCollectionImpl.java:1771)
Caused by: Error : 27486, Position : 0, SQL = begin
  DBMS_SODA_ADMIN.CREATE_INDEX(
                   P_URI_NAME   => :1 ,
                   P_INDEX_SPEC => :2 );
end;, Original SQL = begin
  DBMS_SODA_ADMIN.CREATE_INDEX(
                   P_URI_NAME   => ?,
                   P_INDEX_SPEC => ?);
end;, Error Message = ORA-27486: insufficient privileges

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:710)


Testcase: testGetDescription(oracle.json.tests.soda.test_OracleCollectionAdmin):	Caused an ERROR
null
oracle.soda.rdbms.impl.SODAUtils$1OracleSQLException
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:124)
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:87)
	at oracle.soda.rdbms.impl.OracleDatabaseImpl.callCreatePLSQL(OracleDatabaseImpl.java:2028)
	at oracle.soda.rdbms.impl.OracleDatabaseImpl.createCollection(OracleDatabaseImpl.java:809)
	at oracle.soda.rdbms.impl.OracleDatabaseImpl.createCollection(OracleDatabaseImpl.java:1088)
	at oracle.soda.rdbms.impl.OracleDatabaseImpl.access$200(OracleDatabaseImpl.java:76)
	at oracle.soda.rdbms.impl.OracleDatabaseImpl$OracleDatabaseAdministrationImpl.createCollection(OracleDatabaseImpl.java:2901)
	at oracle.json.tests.soda.test_OracleCollectionAdmin.testGetDescription(test_OracleCollectionAdmin.java:251)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at oracle.json.testharness.JsonTestCase.run(JsonTestCase.java:45)
Caused by: java.sql.SQLException: ORA-28361: Master key not yet set.

https://docs.oracle.com/error-help/db/ora-28361/
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:608)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1277)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1102)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:456)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:482)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:216)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:1164)
	at oracle.jdbc.driver.OracleStatement.executeSQLStatement(OracleStatement.java:1652)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1417)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3731)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3945)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4202)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.execute(OraclePreparedStatementWrapper.java:1011)
	at oracle.soda.rdbms.impl.OracleDatabaseImpl.callCreatePLSQL(OracleDatabaseImpl.java:1964)
Caused by: Error : 28361, Position : 0, SQL = begin
  DBMS_SODA_ADMIN.CREATE_COLLECTION(
                   P_URI_NAME    => :1 ,
                   P_CREATE_MODE => :2 ,
                   P_DESCRIPTOR  => :3 ,
                   P_CREATE_TIME => :4 ,
                   P_23C_DRIVER => true);
end;, Original SQL = begin
  DBMS_SODA_ADMIN.CREATE_COLLECTION(
                   P_URI_NAME    => ?,
                   P_CREATE_MODE => ?,
                   P_DESCRIPTOR  => ?,
                   P_CREATE_TIME => ?,
                   P_23C_DRIVER => true);
end;, Error Message = ORA-28361: Master key not yet set.

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:710)


Testcase: testIsHeterogeneous(oracle.json.tests.soda.test_OracleCollectionAdmin):	FAILED
null
junit.framework.AssertionFailedError
	at oracle.json.tests.soda.test_OracleCollectionAdmin.testIsHeterogeneous(test_OracleCollectionAdmin.java:150)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at oracle.json.testharness.JsonTestCase.run(JsonTestCase.java:45)


